boards/veer_el2_sim/README.md

```diff
@@ -0,0 +1,67 @@
+# VeeR EL2 simulation target
+
+For more information about the core and the Verilator testbench [visit the VeeR EL2 repository](https://github.com/chipsalliance/Cores-VeeR-EL2).
+This was tested with Verilator 5.006, which can be installed as explained [in the doc](https://verilator.org/guide/latest/install.html).
+
+There's also a repository dedicated to [running Tock on VeeR EL2 in simulation](https://github.com/chipsalliance/VeeR-EL2-tock-example)
+
```

Comment:
The previous board had more of an introduction. Can we replicate/preserve that? What type of simulation is this?

Only linking to outside sources makes me nervous. We had that (https://github.com/chipsalliance/Cores-SweRVolf) and now that page does not exist.